code/_helpers/global_lists_vr.dm

```diff
@@ -43,6 +43,13 @@ var/global/list/important_items = list(
 		/obj/item/clothing/suit/storage/hooded/wintercoat/roiz)
 
 var/global/list/digestion_sounds = list(
```

Comment:
Could use a comment explaining this is a list of lists (of sounds)   since it used to just be a list of sounds